packages/commerce-sdk-react/src/hooks/useAuthType.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import useAuth from './useAuth'
+import useLocalStorage from './useLocalStorage'
+
+const onClient = typeof window !== 'undefined'
+
+/**
+ * A hook to return customer auth type, either guest or registered user
+ *
+ */
+const useAuthType = (): string | null => {
```

Comment:
We have a `useCustomerId` hook and I'd love to logically group all auth related utility hook together. One way to do that is to make the names similar. What do you think if we name this `useCustomerType`?

```suggestion
const useCustomerType = (): string | null => {
```